src/api/CoreUtils.ts

```diff
@@ -90,6 +90,11 @@ export class CoreUtils {
         return result;
     }
 
+    public static async getProfileMeta(): Promise<ICommandProfileTypeConfiguration[]> {
+        const ftpProfile = require("../imperative").profiles as ICommandProfileTypeConfiguration[];
+        return ftpProfile;
+    }
```

Comment:
I see the function is `async` but we are not awaiting on the `require`.
I know calling `import(...)` will return a promise, but not 100% sure if async/await is required for `require`.

Reply:
Hey @zFernand0 I had this as async because in the zftp vsc ext code without awaiting this information return the initialization and creation of the zftp profile folder and creation of the meta file would fail

Reply:
got it!
I was just thinking that this may fail some linting rule (`require-await`) in case it gets implemented in the future.

Reply:
ok, let me try with an await in front of require. building and testing full scenario @zFernand0

Reply:
pushed change, this change actually allowed the ftp vsc ext to not have an await on this call and the promise was fulfilled.